config.yaml

```diff
@@ -44,6 +44,13 @@ mongo: mongo-inventory:27017
 # Overwrite with environment variable: INVENTORY_LIMIT_ATTRIBUTES
 # limit_attributes: 100
 
+# Maximum number of seconds between the inventory updates, if
+# you updated the inventory more than this number of seconds it will
+# get updated unconditionally upon next request from a device
+# Defaults to: 86400
+# Overwrite with environment variable: INVENTORY_LAST_UPDATE_SECONDS_THRESHOLD
+# last_update_seconds_threshold: 259200
+
```

Comment:
The default and the value differs.
Also, see :point_up: comment.

Reply:
on purpose :> who says the example should follow the default :>
ok I will move to durations